lib/index.js

```diff
@@ -389,7 +389,10 @@ function handlePropertyValue(schema, value, prop, definition) {
       typeof value === 'number' ||
       typeof value === 'string') &&
     safeProtocol(schema, value, prop) &&
-    (definition.length === 0 || definition.includes(value))
+    (definition.length === 0 ||
+      definition.some((allowed) =>
+        allowed instanceof RegExp ? allowed.test(value) : allowed === value
```

Comment:
I’m not a fan of `instanceof`. It breaks Jest. It breaks across iframes. It breaks across threads.

I prefer ducktyping, something like `allowed && typeof allowed === 'object' && 'flags' in allowed`. Or, invert it: `typeof allowed === 'string'`